Sources/StripeKit/Core Resources/Prices/Price.swift

```diff
@@ -43,6 +43,72 @@ public struct StripePrice: StripeModel {
     public var transformQuantity: StripePriceTransformQuantity?
     /// The unit amount in cents to be charged, represented as a decimal string with at most 12 decimal places.
     public var unitAmountDecimal: Decimal?
```

Comment:
I think we should just make this a `String`.  Stripe objects are annoying and massive and update/change frequently and run the risk of potentially forgetting to add coding keys when things change so I'd like to avoid having to write custom initializers for decoding especially all for just a single property.

Also Stripe specifically returns this value as a `String` to be flexible (also JSON limitations) so I think it makes sense to leave it as the type returned, especially since a user has complete control after that and can parse it however they want.